plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2574,6 +2574,7 @@ elementgui.tool.event_stopped_using=On player stopped using
 elementgui.tool.event_swings=When entity swings item
 elementgui.tool.texture=Tool texture
 elementgui.tool.tool_3d_model=Tool 3D model
+elementgui.tool.shield_blocking_model=Shield blocking model
```

Comment:
Entries on this UI section have colons

![image](https://user-images.githubusercontent.com/16374228/222954486-d172de53-41d8-406c-9f8f-d8f8b9ae293e.png)